docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -155,6 +155,24 @@ The directory to write `yb-tserver` log files.
 
 Default: Same as [`--fs_data_dirs`](#fs-data-dirs)
 
+##### --log_min_seconds_to_retain
+
+The minimum number of seconds for which to keep log segments to keep at all times, regardless of what is required for durability. Logs may be retained for a longer amount of time if they are necessary for correct restart. This should be set long enough such that a tablet server which has temporarily failed can be restarted within the given time period. If a server is down for longer than this amount of time, it is possible that its tablets will be re-replicated on other machines.
+
+Default: `900`
+
+##### --log_min_segments_to_retain
+
+The minimum number of past log segments to keep at all times, regardless of what is required for durability. The value must be at least 1.
+
+Default: `2`
+
+##### --log_segment_size_mb
+
+The default segment size, in megabytes (MB) for log rollovers.
```

Comment:
Not sure if it is clear for a user at this point what a log rollover is? Could maybe clarify that this is (loosely) the max size of each WAL file?

Reply:
Good point -- I'll add that.